idiom-data.js

```diff
@@ -0,0 +1,4 @@
+export const idioms = [{ idiom1: 'Thats the last straw' },
```

Comment:
This could also just be an array of strings and save your function some headache:

`export const idioms = ['thats the last straw', 'miss the boat', 'get your act together', 'a dime a dozen']`

and then you can just loop over idioms and render each string to a `<p>` tag